accounts-db/src/accounts_hash.rs

```diff
@@ -813,59 +824,43 @@ impl<'a> AccountsHasher<'a> {
         (hashes, lamports_total)
     }
 
-    /// returns the item referenced by `min_index`
-    ///   updates `indexes` to skip over the pubkey and its duplicates
-    ///   updates `first_items` to point to the next pubkey
-    /// or removes the entire pubkey division entries (for `min_index`) if the referenced pubkey is the last entry in the same `bin`
-    ///     removed from: `first_items`, `indexes`, and `first_item_pubkey_division`
+    /// Given the item location, return the item in the `CalculatedHashIntermediate` slices and the next item location in the same bin.
+    /// If reaches the end of the `CalculatedHashIntermediate` slice or has exhausted all the accounts in current bin, return None for next item location.
```

Comment:
grammar: If we reach the end... or If the end is reached...

Reply:
Done.